backoffice/backoffice/workflows/tests/test_views.py

```diff
@@ -248,13 +249,32 @@ class TestAuthorWorkflowViewSet(BaseTransactionTestCase):
     reset_sequences = True
     fixtures = ["backoffice/fixtures/groups.json"]
 
-    @patch("backoffice.workflows.airflow_utils.requests.post")
-    def test_create_author(self, mock_post):
-        self.api_client.force_authenticate(user=self.curator)
+    def setUp(self):
+        super().setUp()
+
+        self.workflow = Workflow.objects.create(
+            data={},
+            status="running",
+            core=True,
+            is_update=False,
+            workflow_type="AUTHOR_CREATE",
```

Comment:
please let's use everywhere the constant, to be easy to propagate changes everywhere and to avoid changing strings one-by-one